package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "react-trainings",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --open --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GrigoriyShv/react-trainings.git"
+  },
+  "author": "hshvykyi",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/GrigoriyShv/react-trainings/issues"
+  },
+  "homepage": "https://github.com/GrigoriyShv/react-trainings#readme",
+  "devDependencies": {
+    "babel-core": "^6.26.3",
```

Comment:
By my own experience, I don't recommend you using of tilde and caret for version definition. Because, updating of dependencies can broke your project.
The best practices is manual updating of dependencies